UPGRADE.md

```diff
@@ -722,3 +722,24 @@ If you use complex where condition directives, such as `@whereConditions`,
 upgrade `mll-lab/graphql-php-scalars` to v4:
 
     composer require mll-lab/graphql-php-scalars:^4
+
+### Subscriptions version 1 removal 
+
+Subscriptions only use version 2 now. That means, the extensions content
+will not contain the `channels` key anymore and the `version` will be `2`.
+
+```diff
+{
+  "data": {...},
+  "extensions": {
+    "lighthouse_subscriptions": {
+-      "version": 1,
++      "version": 2,
```

Comment:
I am wondering if we should remove this key altogether.

Reply:
was wondering the same, but this already existed before v2... @stayallive you got an idea about this?

Reply:
Since there are no longer multiple versions (probably was done out of future proofing) I'd say remove it for now, if we need to add it in the future we can always add it back 😄

Reply:
Let's get rid of it. Next time maybe we can name the versions `before` and `after` to make it clear this is temporary.

Reply:
removed it